react-frontend/src/core-components/video/VideoComponent.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
```

Comment:
Please remove this video component from this PR.

1. It needs more work
2. We don't need it now. As per requirement, we can introduce it later.

For dev reference, more details on why it needs more work:

check the props for video component (https://www.w3schools.com/tags/tag_video.asp):
- controls prop takes a string (not object of supported controls like play, pause, etc)
- does not support source as child prop
- autoplay
- poster
to name a few